prepare-solvcon-dev.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
This script uses miniconda for most of the dependencies so I think it's probably good to highlight it in the filename.  We may do it with this PR or act after more discussions.